ui/appui-react/src/appui-react/accudraw/AccuDrawFieldContainer.tsx

```diff
@@ -314,7 +317,11 @@ export function AccuDrawFieldContainer(props: AccuDrawFieldContainerProps) {
           <AccuDrawInputField
             ref={xInputRef}
             isLocked={xLock}
-            className="uifw-accudraw-x-value"
+            className={
+              enableColorlessAccuDrawInputFields
+                ? undefined
+                : "uifw-accudraw-x-value"
+            }
```

Comment:
These are internal classNames, so no consumer should be targeting them (you can resolve this comment). For future reference, in case you'll need to do more styling based on preview features, take a look at `useContentAlwaysMaxSize.scss`, we use https://developer.mozilla.org/en-US/docs/Web/CSS/@layer to figure out what styles take priority.

Reply:
Yea, in my case, it removes the style. 
I dont know how to do that with layers. 
I could do an empty style and overwrite the style with it, but it feels weird to have an empty style. Removing the style felt more intuitive to me.

Reply:
This is fine, let's keep it as is. You can merge this in.